dora-team/dora.dev
# Fix: Sentence Case for Capabilities

√ Fix for https://github.com/dora-team/dora.dev/issues/168

Added sentence case for each capability in the tile. 

<img width="1271" alt="Screenshot 2023-04-10 at 6 18 47 PM" src="https://user-images.githubusercontent.com/10970467/231030425-f32ba19a-25b4-4715-8577-da25955f3f6a.png">


**Steps to Test:**

- From the root directory pls run `hugo serve -D -s hugo --disableFastRender --verbose;`
- Check the changes at `http://localhost:1313/resources/`
